src/components/Navigation.js

```diff
@@ -1,10 +1,10 @@
 function Navigation() {
   return (
     <nav className="navigation">
```

Comment:
[OPTIONAL]
- [ ] Your navigation looks great, however, you're missing the user icon at the top right corner of the design. Please kindly add it

<img width="820" alt="image" src="https://github.com/Diegogagan2587/bookstore/assets/52934128/d19dc4fe-300e-4414-8c6f-da254d275f83">

Reply:
Thanks, Will doit, thanks thanks thanks! :D

Reply:
You're welcome 🙏